README.md

```diff
@@ -187,134 +187,134 @@ We are working hard on getting a full set of module functions on `TaskSeq` that
+| &#x2753;           | `sortBy`             |                      |                                                                   | [note #1](#note1 "These functions require a form of pre-materializing through 'TaskSeq.cache', similar to the approach taken in the corresponding 'Seq' functions. It doesn't make much sense to have a cached async sequence. However, 'AsyncSeq' does implement these, so we'll probably do so eventually as well.") |
+| &#x2753;           | `sortByAscending`    |                      |                                                                   | [note #1](#note1 "These functions require a form of pre-materializing through 'TaskSeq.cache', similar to the approach taken in the corresponding 'Seq' functions. It doesn't make much sense to have a cached async sequence. However, 'AsyncSeq' does implement these, so we'll probably do so eventually as well.") |
+| &#x2753;           | `sortByDescending`   |                      |                                                                   | [note #1](#note1 "These functions require a form of pre-materializing through 'TaskSeq.cache', similar to the approach taken in the corresponding 'Seq' functions. It doesn't make much sense to have a cached async sequence. However, 'AsyncSeq' does implement these, so we'll probably do so eventually as well.") |
+| &#x2753;           | `sortWith`           |                      |                                                                   | [note #1](#note1 "These functions require a form of pre-materializing through 'TaskSeq.cache', similar to the approach taken in the corresponding 'Seq' functions. It doesn't make much sense to have a cached async sequence. However, 'AsyncSeq' does implement these, so we'll probably do so eventually as well.") |
+|                    | `splitInto`          | `splitInto`          |                                                                   | |
+|                    | `sum`                | `sum`                |                                                                   | |
+|                    | `sumBy`              | `sumBy`              | `sumByAsync`                                                      | |
+| &#x2705; [#76][]   | `tail`               | `tail`               |                                                                   | |
+|                    | `take`               | `take`               |                                                                   | |
+| &#x2705; [#126][]  | `takeWhile`          | `takeWhile`          | `takeWhileAsync`, `takeWhileInclusive`, `takeWhileInclusiveAsync` | |
```

Comment:
I usually just put these on new rows (the variants, I mean). Also, is there a reason you changed the column alignment? For the diff, it's probably better not to do that, but perhaps it was necessary?

Also, I'm about to add a full list of all `AsyncSeq` functions covered/not covered, it should then probably go there.

Reply:
Rider did that, and I didnt invest the time to figure out how to tell it not to ;) 
Having takeWhileInclusive on its own line was triggering yet more reformatting
I'll remove the egregious diffs and let you do a more general reorg in due course.